playbooks/upload_to_swift.yml

```diff
@@ -33,24 +80,28 @@
         object_cdn: "{{ service_catalog | selectattr('type', 'equalto', 'rax:object-cdn') | first }}"
       when: rax_pub_cloud
 
-    # os_object does not currently support setting object expiration header field
-    - name: Upload file to Cloud Files
-      command: "swift upload {{ container }} {{ archive_name }} --header 'X-Delete-After:2592000'"
-      args:
-        chdir: "{{ artifacts_dir | dirname }}"
-      no_log: true
-      environment:
-        OS_AUTH_TOKEN: "{{ auth_token }}"
-        OS_STORAGE_URL: "{{ (object_store['endpoints'] | selectattr('region', 'equalto', region) | first)['publicURL'] }}"
+    - set_fact:
+        object_cdn_url: "{{ (object_cdn['endpoints'] | selectattr('region', 'equalto', region) | first)['publicURL'] }}/{{ container }}"
+
+    - set_fact:
+        object_store_url: "{{ (object_store['endpoints'] | selectattr('region', 'equalto', region ) | first)['publicURL']}}"
+
+    - name: Enable CDN
```

Comment:
We should probably have no_log set here to prevent the token leaking into the console log.

Reply:
Done